client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -89,6 +93,14 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
               {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'SIGN IN'}
             </Button>
+            <Button onClick={() => updateLoginContext({
```

Comment:
I guess this is one way of doing it :) 
But we also want a demo user to be created in the database so maybe you can manually create a user, then call the `login` helper function with hardcoded credentials.

Reply:
Haha this is fixed now. User has been created, and will be logged in on click of the "Demo" button